Formula/g/git-spice.rb

```diff
@@ -0,0 +1,33 @@
+class GitSpice < Formula
+  desc "Manage stacked Git branches"
+  homepage "https://github.com/abhinav/git-spice"
+  url "https://github.com/abhinav/git-spice/archive/refs/tags/v0.6.0.tar.gz"
+  sha256 "34b5161ca642bb70269c45b9726137832eb95737465c6134995de5be2e1ef1d6"
+  license "GPL-3.0-or-later"
+  head "https://github.com/abhinav/git-spice.git", branch: "main"
```

Comment:
The following files use the BSD 3 Clause License:
```
internal/komplete/komplete.go
internal/komplete/komplete_test.go
```
We should update the `license` if what we ship includes the functionality in these files.

Reply:
yeah, good call out, updating the license field